x/cronos/spec/04_messages.md

```diff
@@ -0,0 +1,36 @@
+<!-- order: 4 -->
+
+# Messages
+
+## MsgConvertVouchers
+
+Convert native tokens to the mapped CRC20 tokens, if the mapping does not exist and auto-deployment is enabled, an embed CRC20 contract is deployed for it automatically, otherwise, the message fails.
+
++++ https://github.com/crypto-org-chain/cronos/blob/v0.6.0-testnet/proto/cronos/tx.proto#L26-L30
+
+This message is expected to fail if:
+
+- The coin denom is neither IBC nor gravity tokens.
+- The mapping does not exist and auto-deployment is not enabled.
+
+## MsgTransferTokens
```

Comment:
is this the reverse of `MsgConvertVouchers` or how does one "exit" the EVM contract? what if the contract locks the tokens in some way?